engine/src/main/scala/cromwell/engine/workflow/lifecycle/finalization/CopyWorkflowOutputsActor.scala

```diff
@@ -98,7 +98,7 @@ class CopyWorkflowOutputsActor(workflowId: WorkflowId, override val ioActor: Act
     // Truncate regex is declared here. If it were declared in the if statement the regex would have to be
     // compiled for every single file.
     // "execution" should be optional, because its not created on AWS.
-    lazy val truncateRegex = ".*\\/call-.*\\/(execution\\/)?".r
+    lazy val truncateRegex = ".*/call-[^/]*/(execution/)?".r
```

Comment:
NB this folder might also have `attempt-2` (or `-3`, ...) or now `cacheCopy` between the `call-...` and `/execution/`, so we'll probably want to take account of that too